pkg/scheduler/algorithm/priorities/resource_allocation.go

```diff
@@ -96,9 +96,15 @@ func getNonZeroRequests(pod *v1.Pod) *schedulernodeinfo.Resource {
 	result := &schedulernodeinfo.Resource{}
 	for i := range pod.Spec.Workloads() {
 		workload := &pod.Spec.Workloads()[i]
-		cpu, memory := priorityutil.GetNonzeroRequests(&workload.ResourcesAllocated)
-		result.MilliCPU += cpu
-		result.Memory += memory
+		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
```

Comment:
nit: only the GetNonzeroRequests() function is needed under the featureGate()

Reply:
fixed.